hatchery/config.go

```diff
@@ -56,19 +56,29 @@ type AppConfigInfo struct {
 
 // TODO remove PayModel from config once DynamoDB contains all necessary data
 type PayModel struct {
-	Name         string `json:"name"`
-	User         string `json:"user_id"`
-	AWSAccountId string `json:"aws_account_id"`
-	Region       string `json:"region"`
-	Ecs          string `json:"ecs"`
-	VpcId        string `json:"vpcid"`
-	Subnet       int    `json:"subnet"`
+	Id              string  `json:"bmh_workspace_id"`
+	Name            string  `json:"workspace_type"`
+	User            string  `json:"user_id"`
+	AWSAccountId    string  `json:"account_id"`
+	Region          string  `json:"region"`
+	Ecs             string  `json:"ecs"`
+	Subnet          int     `json:"subnet"`
+	HardLimit       float32 `json:"hard-limit"`
+	SoftLimit       float32 `json:"soft-limit"`
+	TotalUsage      float32 `json:"total-usage"`
+	CurrentPayModel bool    `json:"current_pay_model"`
+}
+
+type AllPayModels struct {
+	CurrentPayModel *PayModel  `json:"current_pay_model"`
+	PayModels       []PayModel `json:"all_pay_models"`
 }
 
 // HatcheryConfig is the root of all the configuration
 type HatcheryConfig struct {
 	UserNamespace          string           `json:"user-namespace"`
 	DefaultPayModel        PayModel         `json:"default-pay-model"`
+	DisableLocalWS         bool             `json:"disable-local-ws"`
```

Comment:
this has no use now right?

Reply:
Yeah, it's just a placeholder. We can actually remove it now and re-add later. I never got around to implementing it.